nearcore/src/config.rs

```diff
@@ -433,6 +433,8 @@ pub struct Config {
     /// If set, overrides value in genesis configuration.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub max_gas_burnt_view: Option<Gas>,
+    /// Extra memory database can use for purpose of caches.
+    pub extra_db_memory_bytes: usize,
```

Comment:
I'd do it a little bit different - and create the show "sub struct" that would keep the db settings.

(as we will probably play with other settings too - like what is the size of files, compression etc etc).